voice.go

```diff
@@ -360,6 +360,25 @@ func (v *VoiceConnection) wsListen(wsConn *websocket.Conn, close <-chan struct{}
 				v.wsConn = nil
 				v.Unlock()
 
+				// Wait for VOICE_SERVER_UPDATE.
+				// When the bot is moved by the user to another voice channel,
+				// VOICE_SERVER_UPDATE is received after the code 4014.
+				for i := 0; i < 5; i++ {
+					<-time.After(1 * time.Second)
+
+					v.Lock()
+					reconnected := v.wsConn != nil
+					v.Unlock()
```

Comment:
Since we're only reading the wsConn value, I think it would be appropriate to use R(Un)lock here instead of regular (Un)lock.
```suggestion
					v.RLock()
					reconnected := v.wsConn != nil
					v.RUnlock()
```